### ISSUE TITLE
Can be ViewGroup?

### ISSUE THREAD

**Issue by enriquebautista** [vipulasri/TicketView#12]
How i can obtain this layout with this library?
ImageView with inverter rounded corners or based on shape of layout like view.setClipToOutline(true);
Thanks.
![29](https://user-images.githubusercontent.com/37681056/44688026-87daca80-aa18-11e8-983a-22062948afa4.png)

**Comment by vipulasri** [comment]
What do you mean by inverted rounded corners ? Can show me a sample screenshot/design ?

**Comment by enriquebautista** [comment]
What I want to achieve is the screenshot that I shared. The inverted rounded border is only if you look at the top image. But in itself, what I want to achieve is that the image can have the shape of the TicketView, specifically the arcs of the half, since otherwise the image would be covering half said arcs.

**Comment by vipulasri** [comment]
Have you tried sample app code in the repo, I think I have made a similar kind of view.

**Comment by vipulasri** [comment]
In example activity code, look at the XML how I divided the view. With little inspiration from that you can build this view.

**Comment by enriquebautista** [comment]
yep, but TicketView extends View, and all views draw top of TicketView, and i can't mask the top image. And result in this:
![unnamed](https://user-images.githubusercontent.com/37681056/44694179-a8fbe500-aa31-11e8-93d3-fc1cc3773039.png)

**Comment by vipulasri** [comment]
Okay let me see. what can be done, in this scenario

**Comment by enriquebautista** [comment]
@vipulasri Hi!, any sugguestion?

**Comment by vipulasri** [comment]
@enriquebautista I have to re-write the layout for this work, which will take some time.

**Comment by andreiOliveira** [comment]
this issue have one solution? I need to put background color, like the enriquebautista image, but not working!

**Comment by CoffeeMomoPad** [comment]
Hello @vipulasri, any update on this? I've been trying to pull the Viewgroup conversion off (through extending RelativeLayout) but I'm pretty new in Android (2 months to be exact) so I don't know what I'm missing.
